src/lib/tooltip/tooltip.ts

```diff
@@ -1,46 +1,50 @@
 import {
-    NgModule,
-    ModuleWithProviders,
-    Component,
-    Directive,
-    Input,
-    ElementRef,
-    ViewContainerRef,
-    ChangeDetectorRef
+    NgModule, ModuleWithProviders, Component, Directive, Input, ElementRef, ViewContainerRef,
+    style, trigger, state, transition, animate, AnimationTransitionEvent, NgZone
```

Comment:
These are generally wrapped to be one-per line; WebStorm's `Optimize imports` command should do this automatically (though it always removes the trailing comma and uses 4 spaces).